extensions/spring-data-jpa/deployment/src/main/java/io/quarkus/spring/data/deployment/SpringDataJPAProcessor.java

```diff
@@ -66,6 +80,40 @@ void build(CombinedIndexBuildItem index,
                 interfacesExtendingCrudRepository, indexIndex);
     }
 
+    private void detectAndLogSpecificSpringPropertiesIfExist() {
+        Config config = ConfigProvider.getConfig();
+        Map<String, String> springJpaToQuarkusOrmPropertiesMap = new HashMap<>();
```

Comment:
hi, just curious: why adding them all to a Map? At very least it could be a constant map (static).

One step further, I guess personally I'd not have used a Map at all, just check if any configuration key is equals-to-constant. No need for extensive use of lambdas when a switch on string can do :)

Reply:
Very good points @Sanne!

Reply:
Sometimes I do over engineering :woman_facepalming: :grin: Yes, good points @Sanne thank you. What about this try? Let me know @geoand @gsmet